walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -1,11 +1,11 @@
-:sso-realm-url: https://sso-user-sso.{{openshift-app-host}/auth/admin/{user-username}/console/index.html
+:sso-realm-url: {user-sso-url}/auth/admin/{user-sanitized-username}/console/index.html
```

Comment:
Oh is this a new variable I missed? 👍

Reply:
It's right here ;) 
https://github.com/evanshortiss/integreatly-vscode-asciidoc-snippets/blob/eb9f91c/.vscode/integreatly.code-snippets#L87